_data/wallets.yml

```diff
@@ -34,12 +34,12 @@ websites:
     - name: Bread
       url: https://breadapp.com/
       img: bread.png
-      bcc: Yes
+      bcc: No
```

Comment:
I have the Bread wallet installed on my iOS device. Its a full wallet. Is the online version not fully capable?